src/datatypes/dialogTypes.ts

```diff
@@ -0,0 +1,30 @@
+import { brukertype } from '../types';
+
+export interface Dialog {
+    id: string;
+    overskrift: string;
+    aktivitetId?: string;
+    lest?: boolean;
+    sisteDato: string;
+    sisteTekst?: string;
+    erLestAvBruker?: boolean;
+    venterPaSvar?: boolean;
+    ferdigBehandlet?: boolean;
+    opprettetDato: string;
+    henvendelser?: Array<Henvendelse>;
+    egenskaper?: Array<string>;
```

Comment:
Og her kan man også skrive:
```suggestion
    egenskaper?: string[];
```